project1/application.py

```diff
@@ -33,10 +33,19 @@ def index():
 def register():
     if request.method=="POST" :
         var=request.form.get("email")
-        print(var)
         var1=request.form.get("psw")
-        print(var1)
-        return render_template("gmails.html", gmails=var)
+        timestamp=datetime.datetime.now()
+        user = User(email=var,password=var1,timestamp=timestamp)
+        db.session.add(user)
+        db.session.commit()
```

Comment:
Add the user to the db after confirming they entered both fields ie., in line#48. The idea is to not enter blank information about registered users.

Reply:
ok